Highlighting the downloads links in the nav

jst-cyr opened the pull request:
I'm using this branch for us to try different ways of highlighting the downloads links in the nav, now that the content is in the nav. This first attempt is using simple emoji. It makes them stand out, but doesn't necessarily fit with the rest of the brand style?

Thoughts and suggestions to talk this one through are welcome. I figure this is one of those where we need to see some iterations to see what we want to go with. Not a priority for launch, but starting the discussion while it's in our heads.

robearlam commented:
I like having icons there to show they're a different type of link. The XP & XC save icons work for me.

I'm not sure about the Send & OC elements though, it feels like it works for registration but not for existing users. Maybe we should have 2 links for those products, something like

* Create an account on OC/Send for free
* Log into OC/Send portal

jst-cyr commented:
> I like having icons there to show they're a different type of link. The XP & XC save icons work for me.

What about the icons used for OrderCloud/Send? 

> I'm not sure about the Send & OC elements though, it feels like it works for registration but not for existing users. Maybe we should have 2 links for those products, something like
> 
> * Create an account on OC/Send for free
> * Log into OC/Send portal

I'm fine with us updating the nav to have more elements, but for this particular PR we should contain to the highlighting portion. Unless you are meaning that would impact how you would want those highlighted if there were now two links showing?

**NOTE:** I also tried using the SVG that is in 'waffle' nav menu at the top to indicate that it goes somewhere else, but I lacked the CSS skills to get it to line up appropriately, just putting that out there as another option to show that something 'different' is there.

markvanaalst reviewed (commented):
Personally I would use one link for both Moosend and OrderCloud calling it something like "Try OrderCloud out right now". Both login pages mention people can create a account for free, so it seems overhead of adding that to our menu.

As for the icons; I would use something more "static" like an arrow pointing to the right indicating that its an external link. (but that is a personal preference)

robearlam commented:
Yeah, I'm fine with combining the two if thats what people prefer. If we do combine them, then I think using a static arrow like @markvanaalst suggested makes more sense.

jst-cyr commented:
I've updated with the latest from Main, merged the OrderCloud and Send back together into a single link, and added an emoji 'arrow'. Is this the idea folks were thinking might work?

Or do we want to use a non-emoji arrow like the one in the Waffle menu?

robearlam commented:
I think I prefer using the same SVG as used in the burger menu. I just submitted a change where you can now mark a link as being `external` and the nav component will render the SVG with it.

What do you guys think of this approach?

jst-cyr commented:
I like what is here, but since it's my PR, I can't "approve" this review. I want to say that in dark mode, the icon used by Rob looks better, but the one Mark has is consistent with the waffle menu icons. Let's go with that for now.

@markvanaalst or @robearlam , whomever sees this first, can you do the final "approval" and then merge to main?
